fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -273,6 +326,65 @@ impl From<ConsensusParametersV1> for ConsensusParameters {
     }
 }
 
+/// A collection of parameters for convenience
+/// The difference with [`ConsensusParametersV1`]:
+/// - `block_transaction_size_limit` has been added.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, serde::Serialize, serde::Deserialize)]
+pub struct ConsensusParametersV2 {
+    pub tx_params: TxParameters,
+    pub predicate_params: PredicateParameters,
+    pub script_params: ScriptParameters,
+    pub contract_params: ContractParameters,
+    pub fee_params: FeeParameters,
+    pub chain_id: ChainId,
+    pub gas_costs: GasCosts,
+    pub base_asset_id: AssetId,
+    pub block_gas_limit: u64,
+    pub block_transaction_size_limit: u64,
```

Comment:
This is not related to this PR only, but in general I think we should strive to have as much documentation as possible, including individual fields. 

For example in this case it is not 100% clear if `block_transaction_size_limit` refers to the number of transactions in a block, or the number of bytes in a transaction.